src/test/java/Calculator/ScientificCalculatorTest.java

```diff
@@ -0,0 +1,19 @@
+package Calculator;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ScientificCalculatorTest {
+
+    @Test
+    public void ScientificCalculatorTest(){
+        ScientificCalculator scientificCalc = new ScientificCalculator();
+        double a = 6;
+        double b= 3;
```

Comment:
please use formatter, your code is not formatted

Reply:
Done, added Save Actions plugin.